### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by foadlind**
Update changelog with unreleased changes after version 0.1.1.

### CODE REVIEW

**Comment by fghaas** [comment]
Please review the alternate proposal. :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## Unreleased
+ 
+* [Fix] Rather than incorrectly relying on `ENABLE_HTTPS`, honour
+  `S3_USE_SSL` when setting `AWS_S3_ENDPOINT_URL`.
+* [Fix] Only populate the `AWS_S3_ENDPOINT_URL` Open edX setting if
+  `S3_ENDPOINT_URL` is set, and recommend leaving `S3_ENDPOINT_URL`
```
Comment (foadlind) [inline CHANGELOG.md:6]: The user doesn't directly set `S3_ENDPOINT_URL` they set `S3_HOST` and `S3_PORT` instead and we set the`AWS_S3_ENDPOINT_URL` for them.

**Review by foadlind — COMMENTED** [review]
Looks better :) I have one comment.

File: CHANGELOG.md (hunk 1 above)
Comment (foadlind) [reply CHANGELOG.md:6]: I can update the text.

Comment (fghaas) [reply CHANGELOG.md:6]: Ah yes, fair point. Please do. :)

Comment (foadlind) [reply CHANGELOG.md]: Done

**Pull request merged**